odoo/addons/base/models/res_users.py

```diff
@@ -359,10 +359,15 @@ def _compute_tz_offset(self):
 
     @api.depends('groups_id')
     def _compute_accesses_count(self):
+        IrRule = self.env['ir.rule']
```

Comment:
if used only once, no need to make a variable for this ;) 
while we are at it, we could externalize this in a method _getRules or something, as it's used twice in same file.

Reply:
@dbeguin done